lib/si-test-macros/src/expand.rs

```diff
@@ -47,7 +44,12 @@ pub(crate) fn expand_test(item: ItemFn, _args: Args, fn_setup: impl FnSetup) ->
         #(#attrs)*
         fn #test_name() -> ::dal_test::Result<()> {
             use ::dal_test::WrapErr;
+            use ::std::io::Write;
+            use ::dal_test::telemetry::tracing;
 
+            // Could add a custom `name` attribute here to send through the test name
+            // SI_TEST_LOG=test_integration::integration_test::rebaser=info,off SI_TEST_LOG_SPAN_EVENTS=new,close buck2 run lib/dal:test-integration -- integration_test::rebaser --nocapture
+            #[tracing::instrument(level = "info", name="example-test-name", skip_all)]
```

Comment:
Should we keep `example-test-name` or am I not understanding what this does?

Reply:
It's a placeholder for when we figure out how to pipe through the test name into the macro, this is what the otel span has as the identifier attribute. Ideally it would be the test file path/name that is pushing the trace up but because I was not receiving the traces I had no way to actually validate any logic above was correctly setting the name (if that makes sense)